### PR TITLE
Add a menu item to delete both local and remote branch at once

### PR DESCRIPTION
**Pull request by stefanhaller**
- **PR Description**

Add a menu item to delete both local and remote branch at once.

- **Please check if the PR fulfills these requirements**

* [x] Cheatsheets are up-to-date (run `go generate ./...`)
* [x] Code has been formatted (see [here](https://github.com/jesseduffield/lazygit/blob/master/CONTRIBUTING.md#code-formatting))
* [x] Tests have been added/updated (see [here](https://github.com/jesseduffield/lazygit/blob/master/pkg/integration/README.md) for the integration test guide)
* [x] Text is internationalised (see [here](https://github.com/jesseduffield/lazygit/blob/master/CONTRIBUTING.md#internationalisation))
* [ ] If a new UserConfig entry was added, make sure it can be hot-reloaded (see [here](https://github.com/jesseduffield/lazygit/blob/master/docs/dev/Codebase_Guide.md#using-userconfig))
* [ ] Docs have been updated if necessary
* [x] You've read through your own file changes for silly mistakes etc

### CODE REVIEW

File: pkg/gui/controllers/helpers/branches_helper.go [hunk 1]
```diff
@@ -97,6 +97,59 @@ func (self *BranchesHelper) ConfirmDeleteRemote(remoteName string, branchName st
...
+	)
+
+	if !isMerged {
+		prompt += "\n\n" + utils.ResolvePlaceholderString(
+			self.c.Tr.ForceDeleteBranchMessage,
+			map[string]string{
+				"selectedBranchName": branch.Name,
+			},
+		)
+	}
```
Comment (stefanhaller) [inline pkg/gui/controllers/helpers/branches_helper.go:126]: I decided to put up only a single confirmation ("are you sure you want to delete both branches"). If the local branch is not merged, I append the warning about this below in the same popup. I like this better than putting up a separate popup for this after the first one, but I can change it if you disagree.

**Review by jesseduffield — APPROVED** [review]
LGTM, great work. I agree about combining the messages: it makes for fewer keypresses

**Comment by philbarto** [comment]
Hey @stefanhaller this may be a stupid question but you just press `b` when on a branch you're not checked out on to delete the local and remote branch, correct? That's what I could gather by reading the code but that doesn't do anything on my machine.

**Comment by philbarto** [comment]
@mark2185 Yeah that's what I was thinking too, although I don't see the `b` option when trying to delete a branch

Screenshot 2024-10-24 at 12 20 50 PM

**Comment by mark2185** [comment]
@philbarto can't check exact dates on my phone, but check if this wasn't merged after `v0.44.1` was released.

If you can build `master` from source, you should have it.

**Pull request merged**